backend/de.metas.business/src/main/java/de/metas/pricing/service/ProductPrices.java

```diff
@@ -143,6 +143,8 @@ public static void assertUomConversionExists(final I_M_ProductPrice productPrice
 		final IProductDAO productDAO = Services.get(IProductDAO.class);
 		final org.compiere.model.I_M_Product product = productDAO.getById(productPrice.getM_Product_ID());
 
+		if(product.isSkipPriceUOMValidation()){return;}
```

Comment:
what's that?

Reply:
In modular contracts we have Computing Methods that have products with stockUOM PCE and are displayed in Invoice with 1 PCE always, but on log calculation we need a price with weightUOM for calculation.
As there is no conversion between these 2 UOMs in this case that makes sense, for these products the validation can be deaktivated.
As the price needs to be added before the product is used, I don't see a reliable way to check it automatically